test/test.py

```diff
@@ -668,6 +668,17 @@ def assert_bundle():
 
         assert_bundle()
 
+    def test_file_core(self):
+        uuid = '70184761-70fc-4b80-8c48-f406a478d5ab'
+        version = '2018-09-05T182535.846470Z'
+        manifest, metadata_files = self._load_bundle(uuid, version, replica='aws', deployment='staging')
+        bundle = Bundle(uuid, version, manifest, metadata_files)
+        file_names = {file.manifest_entry.name for file in bundle.files.values()}
+        formats = {file.format for file in bundle.files.values()}
+        content_descriptions = {file.content_description for file in bundle.files.values()}
+        self.assertEqual(file_names, {'R1.fastq.gz', 'R2.fastq.gz'})
```

Comment:
Let's try not to add more tests that index the same bundle just to assert different aspects of that bundle. Follow the pattern that's already established or indicate why that's not possible here.

Reply:
My mistake. I thought we were trying to move away from enlarging `_assert_bundle` with each new field addition.

Reply:
No, you are right. We started to move to more specialized test cases and we were able to do so because we started to used canned bundles which speed up the tests. But my error is blessing in disguise because it is nice to have a test with a broader set of assertions on a more recent bundle.